tutorials/enricher/src/site/conf/my-app-conf1.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Its part of standard template of apex app archetype. Hence kept it as it is. All other examples have also done the same.